docs/patterns/clustered-listening-tentacles.md

```diff
@@ -0,0 +1,144 @@
+ - Configured the new instance to listen on TCP Port 10933 and set the Application and Home directories to our Shared Storage drive.
+ - Configured the tentacle to trust our Octopus Server holding a certificate which matches the Certificate Thumbprint
+ - Ensured the Windows Firewall has a rule configured to allow incoming connections on TCP Port 10933 allowing the Octopus Server to talk to our new Tentacle.
+
+Using the Tentacle Manager stop the Octopus Tentacle service which was just installed on the first node and take the shared disk offline in Windows Disk Management. 
+
+Now go to the second tentacle server in the active/passive cluster and bring the same disk online, repeating the steps which were just performed on the first node to install the Octopus Tentacle service. Please keep the Octopus Tentacle service started and ensure that the Shared Storage is still mounted this time so that the .pfx file can be exported out of Octopus Tentacle.
+
+## Generate an Octopus Tentacle PFX File {#ClusteringTentacles-Newpfx}
+
+Open `cmd` as Administrator again and run these commands to generate a new Private Key from an Octopus Deploy Tentacle (replacing relevant values as appropriate).
+```batchfile
+cd "C:\Program Files\Octopus Deploy\Tentacle\"
+Tentacle.exe new-certificate --export-pfx="<sharedStorageDriveLetter>:\TentacleClusterPrivateKey.pfx" --pfx-password="Yourpfxpassword"
+```
+
+## Import the new Octopus Tentacle PFX File {#ClusteringTentacles-Importpfx}
+
+Now import the new pfx file into the server from which it was just generated.
+```batchfile
+Tentacle.exe import-certificate --instance="Tentacle" --from-file="<sharedStorageDriveLetter>:\TentacleClusterPrivateKey.pfx" --pfx-password="Yourpfxpassword"
+Tentacle.exe service --instance="Tentacle" --stop --start
+```
+
+Now on the second node, stop the Tentacle service and bring the shared storage offline the same way you have done before. This time go back to the first node in the cluster, bringing the Shared Storage drive back online and start the tentacle service. Then, import the pfx file into the first node to ensure both nodes in the cluster hold the same private key. 
+
+Once both Tentacles are installed and configured ensure that neither node has the Octopus Tentalce started and that the shared storage is brought offline.
+
+## Configure A new Clustered Service {#ClusteringTentacles-NewCluster}
+
+Ensure each node that will be participating in the Tentacle Cluster is joined to the Active Directory Domain and has the **Failover Clustering** feature installed in Windows. For more information on installing the Failover Clustering feature in windows follow [this link](https://blogs.msdn.microsoft.com/clustering/2012/04/06/installing-the-failover-cluster-feature-and-tools-in-windows-server-2012/ "installing the failover cluster service feature and toold in windows server 2012")
```

Comment:
Never use `this link` or `click here` for a link. Its bad for accessibility. Try something like `For more information on installing the Failover Clustering feature in windows please see the  [Microsoft Failover Clustering documentation]`.
Also missing a trailing `.`.